### DISCUSSION TITLE
how to avoid dynamic hashes in font-family using Next local font

### DISCUSSION THREAD

**Discussion by smo043** [vercel/next.js#49257] [App Router]
I'm using Next Js 13.4 and have utilised the localFont functionality to place all of my local font files inside the app directory in a fonts folder. However I discovered that generated file layout.CSS has hashed font-family, which is causing fonts on the page to break. I tried several online alternatives to override hash in next config webpack, but none of them worked. Looking for assistance from this group. I want the font family to be "MyFont"

This is how hashed fontFamily is created "font-family: '__MyFont_f77142';"

**Comment by max-slieptsov** [comment]
@smo043 have you found a way to fix it?

**Reply by smo043** [reply]
Yes I am able to resolve it for my custom fonts

**Reply by max-slieptsov** [reply]
could you please share fix?

**Reply by smo043** [reply]
> could you please share fix?

Here are my contribution links, hope these are helpful.

https://github.com/mui/material-ui/pull/37315

https://github.com/mui/material-ui/pull/37315#pullrequestreview-1438977591

**Reply by max-slieptsov** [reply]
Thank you

**Reply by smo043** [reply]
> Thank you

Was it helpful?
